### PR TITLE
Adds egmont reference text

### PR DESCRIPTION
**Pull request by furaone**
Separates body footer styles from other footers
Adds styles for blockquote tag
image

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -370,22 +370,33 @@ section {
...
+body > footer p {
     font-size: 0.75em;
 }
 
+blockquote p {
+    border-left: 0.25em solid rgba(255, 255, 255, 0.25);
+    padding-left: 0.5em;
+}
+
+blockquote > footer {
```
Comment (Keraxel) [inline style.css]: Does it have to be a direct descendant? Seems inconsistent with `blockquote p`

File: style.css [hunk 2]
```diff
@@ -370,22 +370,33 @@ section {
     -webkit-transform: translateX(-50%);
 }
 
-footer {
+body > footer {
```
Comment (Keraxel) [inline style.css]: Wouldn't it be better to create a class instead of creating direct descendant hierarchy?

File: index.html [hunk 3]
```diff
@@ -114,9 +114,12 @@ <h2 class="h1">Our Clients</h2>
...
                 </div>
                 <div class="c-client__description">
-                  <p>We cooperate with Egmont publishing since a year. Together we released <a href="https://talk-town.dk" target="_blank">Talk Town</a> podcast application.
-                        During this project an iOS, Android, web application and CMS was created.
-                    </p>
+                    <blockquote>
+                        <p>Nephilim is one of our most trusted technological partners. They deliver and develop solutions across the whole stack. They always deliver on time, in great quality and we are always getting a lot of constructive suggestions on complex solutions from them. Over the past year we have, among other things, developed two native mobile apps and built strong business relationship at the same time.
+                            We feel safe having such skilful and trustworthy partner at hand.</p>
+
+                        <footer>- Mads, Head of Digital Development</footer>
```
Comment (YANOUSHek) [inline index.html]: Replace `-` with `–`

```suggestion
                        <footer>– Mads, Head of Digital Development</footer>
```

File: style.css (hunk 1 above)
Comment (furaone) [reply style.css]: added `blockquote > p`

File: style.css (hunk 2 above)
Comment (furaone) [reply style.css]: It would be better in the case when you are building application - here use classes only when there is no other solution.

**Pull request merged**